Source/Blazorise/Blazorise.csproj

```diff
@@ -23,4 +24,44 @@
     <EmbeddedResource Include="Resources\Localization\*.json" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Remove="bundleconfig.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="Scripts\blazorise.dependencies.js" />
+    <None Remove="Scripts\blazorise.functions.js" />
+    <None Remove="Scripts\blazorise.main.js" />
+    <None Remove="Scripts\blazorise.table.js" />
+    <None Remove="Scripts\blazorise.tooltip.js" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="Scripts\blazorise.table.js">
```

Comment:
Instead of naming every file you can just use a wildcard.

Reply:
You're right. It's just automatic from vs ide.